cms/drupal/db/backup/create.php

```diff
@@ -1,26 +1,34 @@
 <?php
 
+/**
+ * Create a drupal database
+ *
+ * @file
+ */
+
+declare(strict_types=1);
+
 namespace Deployer;
 
-task('cms:drupal:db:backup:create', function () {
+task('cms:drupal:db:backup:create', static function (): void {
   // Ensure that the backup directory exists.
-  run('mkdir -p {{backups}}/latest');
+    run('mkdir -p {{backups}}/latest');
 
   // Move files out of the "latest" backups folder.
-  run('[ "$(ls -A {{backups}}/latest)" ] && mv -v {{backups}}/latest/* {{backups}} || echo "No backups found in latest folder."');
+    run('[ "$(ls -A {{backups}}/latest)" ] && mv -v {{backups}}/latest/* {{backups}} || echo "No backups found in latest folder."');
 
   // Create the backup file.
-  within(get('app_path'), function () {
-    $date = date('Y-m-d--H-i-s');
-    // TODO: support exporting all sites found in site aliases list.
-    // $result = run("{{drush}} sa --format json");
-    // if (empty($result)) {
-    //   throw new \UnexpectedValueException('No site aliases found!');
-    // }
-    // $aliases = json_decode($result);
-    //foreach ($aliases as $alias => $info) {}
+    within(get('app_path'), static function (): void {
+        $date = \date('Y-m-d--H-i-s');
+      // TODO: support exporting all sites found in site aliases list.
+      // $result = run("{{drush}} sa --format json");
+      // if (empty($result)) {
+      //   throw new \UnexpectedValueException('No site aliases found!');
+      // }
+      // $aliases = json_decode($result);
+      //foreach ($aliases as $alias => $info) {}
 
-    $filename = "{{application}}--$date.sql";
-    run("{{drush}} rq && {{drush}} sql:dump --gzip --result-file={{backups}}/latest/$filename || echo 'Database not available'", ['timeout' => NULL]);
-  });
+        $filename = '{{application}}--' + $date + '.sql';
+        run('{{drush}} rq && {{drush}} sql:dump --gzip --result-file={{backups}}/latest/' + $filename + "|| echo 'Database not available'", ['timeout' => null]);
```

Comment:
This should probably be `.` instead of `+`.  Or alternatively, you could use `\sprintf()`.

Reply:
Good catch!